packages/fxa-content-server/server/lib/csp/blocking.js

```diff
@@ -37,6 +37,7 @@ module.exports = function(config) {
   const PAIRING_SERVER_HTTP = PAIRING_SERVER_WEBSOCKET.replace(/^ws/, 'http');
   const SENTRY_SERVER = 'https://sentry.prod.mozaws.net';
   const SURVEY_GIZMO = 'https://qsurvey.mozilla.com';
+  const SURVEYS = config.get('surveys').map((s) => s.url).map(getOrigin);
```

Comment:
I think this would produce an array of duplicate origins when the survey urls have the same origin.  I don't know if this will be an issue later when the array is used.

Reply:
Good point, should check for that